scripts/fast-reboot

```diff
@@ -272,13 +273,45 @@ function backup_database()
     fi
 }
 
+function check_mirror_session_acls()
+{
+    ACL_ARP="missing"
+    ACL_ND="missing"
+    start_time=${SECONDS}
+    elapsed_time=$((${SECONDS} - ${start_time}))
+    while [[ ${elapsed_time} -lt 10 ]]; do
+        ACL_ENTRIES=( $(redis-cli -n 1 KEYS "*" | grep SAI_OBJECT_TYPE_ACL_ENTRY) )
```

Comment:
As we found out in the test - this step is problematic. Please add a way to handle the case when no ACL_ENTRY objects are found, and grep fails with non-zero exit code. 

The goal is for any failure we see here, the error should be gracefully handled and exit code should be `EXIT_NO_MIRROR_SESSION_ACLS`

Reply:
Wouldn't your latest change also fail when grep does not find anything?  In the current code if grep fails then you wouldn't move to next step to check `$?`, rather the code will just exit from this line.

Example code below. But please test it out.

```bash
CHECK_ACL_ENTRIES=0
ACL_OUTPUT=$(sonic-db-cli ASIC_DB KEYS "*" | grep SAI_OBJECT_TYPE_ACL_ENTRY) || CHECK_ACL_ENTRIES=$?
if [[ CHECK_ACL_ENTRIES-ne 0 ]]; then
      error "Failed to retrieve SAI_OBJECT_TYPE_ACL_ENTRY from ASIC_DB"
fi
# Convert output to list
ACL_ENTRIES=( ${ACL_OUTPUT} )
```